src/lib/Libwin/passwd.c

```diff
@@ -4817,15 +4817,15 @@ logon_pw(char *username,
 	char msg[4096])
```

Comment:
It can be declared as character pointer, but can also be passed as a sized array; it will be converted to a character array. Should not be a problem.